src/main/haskell/kore/src/Kore/Builtin/String.hs

```diff
@@ -380,6 +385,27 @@ evalString2Int = Builtin.functionEvaluator evalString2Int0
                 . fmap fst . listToMaybe . readSigned readDec
                 $ _str
 
+evalChr :: Builtin.Function
+evalChr = Builtin.functionEvaluator evalChr0
+    where
+    evalChr0
+        :: Ord (variable Object)
+        => MetadataTools Object StepperAttributes
+        -> StepPatternSimplifier Object variable
+        -> Sort Object
+        -> [StepPattern Object variable]
+        -> Simplifier (AttemptedFunction Object variable)
+    evalChr0 _ _ resultSort arguments =
+        Builtin.getAttemptedFunction $ do
+            let _n =
+                    case arguments of
+                        [_n] -> _n
+                        _    -> Builtin.wrongArity lengthKey
+            _n <- Int.expectBuiltinInt chrKey _n
+            Builtin.appliedFunction
+                . asExpandedPattern resultSort
+                $ [ toEnum (fromIntegral _n) ] 
```

Comment:
Minor nitpick: I would find it easier to read this line if it used `chr` (from `Data.Char`) instead of `toEnum`, but I don't feel too strongly about it.

Reply:
I agree, `chr` is definitely easier to read then `toEnum`. I am going to fix this now.